client/src/lib/Validator.js

```diff
@@ -35,6 +35,10 @@ class Validator {
   }
 
   validateValue(value, rule, config) {
+    // Empty values suppress error unless rule is required
+    if (value === '') {
+      return rule !== 'required';
+    }
```

Comment:
neat... but check out `validateField` line 157 and line 174, wouldn't that have caught the `required` rule already?

Maybe that part could be investigated to be removed and shifted here

Reply:
That line only shortcuts required failures (required = true, empty = true).

This fix shortcuts non-required successes (required = false, empty = true) which should bypass other rules, but doesn't. Thus all of the specicchecks.validate('') return false.